content/reference/lattice-protocols/rpc.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "RPC"
+date: 2018-12-29T11:02:05+06:00
+weight: 2
+draft: false
+---
+
+At its core, the _lattice_ is essentially an RPC bus layer built on top of the NATS message broker. When the lattice
+is enabled, participants in lattice RPC must conform to a set of standards in order to be able to send and handle messages.
```

Comment:
For clarity, perhaps either the title or first mention of RPC should be `Remote Procedure Call (RPC)`